src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -33,9 +33,10 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <!-- Default XPath transforms for bindings -->
     <TransformFile Include="Transforms\**\*.xml" />
     <!-- Default Java or native libraries -->
-    <AndroidLibrary       Include="**\*.jar" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <AndroidLibrary       Include="**\*.jar" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);**\*-source?.jar;**\*-src.jar" />
```

Comment:
What does the `?` in `**\*-source?.jar` do?

Reply:
It's a [filesystem glob](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-items?view=vs-2019#use-wildcards-to-specify-items); `?` matches zero or one characters.

In this case, it's to match `*-sources.jar` (plural) and `*-source.jar` (singular).  (And if someone has a `*-sourcez.jar`, that'll be grabbed too, arguably inadvertently, but I think that's highly unlikely.)

Reply:
…or I misread! Looks like MSBuild documents `?` as "a single character", not zero-or-one.  Guess I need to be more explicit then. :-/

Reply:
The globs have been moved into a new `$(_DefaultJavaSourceJarPattern)` property, which is used in both Include & Exclude, as required.